packages/zoe/src/contracts/sellItems.js

```diff
@@ -103,7 +103,7 @@ const start = zcf => {
     buyerSeat.exit();
 
     if (itemsMath.isEmpty(publicFacet.getAvailableItems())) {
-      zcf.shutdown();
+      zcf.shutdown('All items sold.');
```

Comment:
nit: grammar of the `reason` is inconsistent with the one in `atomicSwap.js`, if we're establishin a convention here we that people are going to learn from, we might want to pick a consistent one.

Reply:
I'm not sure I get your point.  atomicSwap is a contract that does a single exchange, and then it's done and can't be used again. sellItems has a list of items to sell, and it remains open until all the items have been sold. They each announce the condition that caused them to shut down.  In what way do they seem inconsistent to you?

Reply:
Trivial, but `All items sold.` is a capitalized phrase with a terminating period, whereas atomicSwap.js says `'swap completed'` which is not capitalized and lacks the period.